runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -397,6 +399,10 @@ public void onChatMessage(ChatMessage event)
 					log.warn("Unreachable default case for message ending in '; return to Slayer master'");
 			}
 			setTask("", 0);
+			if (config.maximumPointsNotification() && Integer.toString(streak).endsWith("9"))
```

Comment:
Add empty line above, also do what @Polar said, do not convert the number back to string second time

Reply:
Excuse me, what do you mean "do not convert the number back to string second time"

Reply:
use modulo

Reply:
Few lines above it is converted from string to number and then you are converting it back to string